spdx/snippet.py

```diff
@@ -49,6 +54,8 @@ def __init__(
         self.snip_from_file_spdxid = snip_from_file_spdxid
         self.conc_lics = conc_lics
         self.licenses_in_snippet = []
+        self.byte_range: Tuple[Optional[int], Optional[int]] = byte_range
+        self.line_range: Optional[Tuple[Optional[int], Optional[int]]] = None
```

Comment:
Both of the types should be `Optional[Tuple[int, int]]`, and `byte_range` should be initialized as `None`

Reply:
done

Reply:
Types are still not adjusted (I am not even sure what `Optional[Optional[int], Optional[int]]` means :thinking: )